nnpackage/examples/README.md

```diff
@@ -1,5 +1,12 @@
 # NNPackage example
 
+## Package version 1.2.0
+
+### one_op_in_bin
+
+- Model file: pre-compiled dynamic library (TBD)
+- Only one `ADD` operation (TBD)
```

Comment:
Since it is pre-compiled binary, we do not need to specify binary's internal operations ? IMHO, this binary will be loaded into Graph of single operation.

Reply:
Right. Internal operation is not important. It's just explain example model.

Reply:
```suggestion
- Compiled model has only one `ADD` operation (TBD)
```